### PR TITLE
Allow extendable classes in FormModal

### PR DESCRIPTION
**Pull request by jfurrow**
This will allow us to extend the `FormModal`'s content classes, which I need for an upcoming PR.

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @MatApple

File: src/js/components/FormModal.js [hunk 1]
```diff
@@ -168,6 +173,11 @@ FormModal.defaultProps = {
 FormModal.propTypes = {
   buttonDefinition: React.PropTypes.array,
   children: React.PropTypes.node,
+  contentClasses: React.PropTypes.oneOf([
+    React.PropTypes.array,
+    React.PropTypes.string,
```
Comment (mlunoe) [inline src/js/components/FormModal.js]: #nit :abc:

**Comment by mlunoe** [comment]
Looks good! :shipit:

File: src/js/components/FormModal.js (hunk 1 above)
Comment (jfurrow) [inline src/js/components/FormModal.js]: Done!

**Pull request merged**